apps/api/src/schema/websoc.ts

```diff
@@ -155,6 +155,24 @@ export const websocQuerySchema = z.object({
       }
       return parsedCodes;
     }),
+  includeRelatedCourses: z
+    .string()
+    .optional()
+    .transform((value, ctx) => {
+      if (!value) return false;
```

Comment:
And check for `"undefined"` in this function (I think that's how `.coerce` works...)